### ISSUE TITLE
empty # FIXME is ignored by mix credo

### ISSUE THREAD

**Issue by pdobacz** [rrrene/credo#585]
### Environment

* Credo version (`mix credo -v`): 0.10.0
* Erlang/Elixir version (`elixir -v`): 
```
Erlang/OTP 20 [erts-9.3.3.3]
Elixir 1.7.2 (compiled with Erlang/OTP 20)
```
* Operating system: Ubuntu 16.04

### What were you trying to do?

`mix credo -a` on a file with `# FIXME` (i.e. no "contents")

### Expected outcome

```
┃ [D] ↗ Found a FIXME tag in a comment: # FIXME
```

`credo` should report the fixme regardless of the lack of comment. This is useful where the next line makes it evident, what's there to fix

### Actual outcome

`credo` doesn't mind.

**Comment by rrrene** [comment]
Hi @pdobacz, thx for this issue.

The current behaviour is by design and works as expected. The thinking behind it was that a `FIXME` comment should always include some text, i.e. the first steps toward fixing it.

Examples:

```elixir
# FIXME: use Base58
# FIXME: include user's last name
# FIXME: rescue this exception
```

WDYT?

**Comment by OvermindDL1** [comment]
I'm definitely +1 on forcing FIXME's and TODO's to have a trailing text as it makes `grep`ping for them far easier to discern!

**Comment by pdobacz** [comment]
Hi @rrrene, thanks! If that's by design then OK, I will conform. Feel free to close this.

**Comment by rrrene** [comment]
@pdobacz :+1:
